src/bin/util/setup.rs

```diff
@@ -29,6 +29,12 @@ use tikv::util::logger;
 
 // A workaround for checking if log is initialized.
 pub static LOG_INITIALIZED: AtomicBool = ATOMIC_BOOL_INIT;
+// Default is 128.
+// Extended since blocking is set, and we don't want to block very often.
+const SLOG_CHANNEL_SIZE: usize = 10240;
+// Default is DropAndReport.
+// It is not desirable to have dropped logs in our use case.
+const SLOG_CHANNEL_OVERFLOW_STRATEGY: OverflowStrategy = OverflowStrategy::Block;
```

Comment:
Does the strategy affect performance?

Reply:
If we overflow the channel it will start to block, so yes.

Reply:
I browsed the slog-async code, if we choose to drop the message, it will still log an error which may still block the process if I/O is busy...